### PR TITLE
Avoid DCM functionality from maintenance manager module

### PR DESCRIPTION
**Pull request by V3110**
Reason for change: DCM exectution removed from MM as DCM is starting from service seperately
Test Procedure: NA
Risks: None
Priority: P2

### CODE REVIEW

File: amlogic.cmake [hunk 1]
```diff
@@ -165,3 +165,5 @@ endif()
 if(BUILD_ENABLE_APP_CONTROL_AUDIOPORT_INIT)
    add_definitions (-DAPP_CONTROL_AUDIOPORT_INIT)
 endif()
+
+add_definitions (-DDCM_REMOVAL)
```
Comment (PaviP98) [inline amlogic.cmake:169]: Please check whether it is DCM_REMOVAL or DCM_REMOVAL_MM

Comment (V3110) [reply amlogic.cmake]: As discussed, modified the flag to DCM_TASK_REMOVAL. 
Validated the functionality. Working fine.

**Review by PaviP98 — APPROVED** [review]
LGTM

**Review by tdeva14 — APPROVED** [review]
LGTM

**Review by anand-ky — APPROVED** [review]

**Pull request merged**